### DISCUSSION TITLE
申请退款，谢谢！

### DISCUSSION THREAD

**Discussion by YukieWu** [getlantern/lantern-forum-cn#539] [General]
您好Mike，我于今年三月订阅了一年的蓝灯（邮箱：824337028@qq.com）。非常感谢蓝灯团队的技术支持。

但是连接实在太不稳定，几个月里能连上的时间不多且慢；重要功能受限，且苹果系统始终没有pro版。

功能没有办法我正常使用网路。所以请帮忙申请取消订阅，并将剩下的费用退还。谢谢！

**Comment by MaxMKMax** [comment]
@YukieWu 您好. 请您也提供我支付单据. 我会帮您申请. 退款一般需要 14-21 天来完成. 谢谢!

**Reply by songguojian0720** [reply]
你好，我的账号现在几乎是无法使用，网超级差，图片也打不开，去年买的是两年的，现在申请退款剩下一年的。请帮忙处理下

**Reply by YukieWu** [reply]
@Mike777Khoo 
你好Mike，之前支付单据已通过邮箱发送给你。请问退款进度到哪里了呢？
祝好。

**Comment by YukieWu** [comment]
你好，感谢回复！

附件是收据，请查收

Mike K ***@***.***> 于 2024年6月2日周日 13:57写道：

**Comment by YukieWu** [comment]
Hi Mike,

不好意思又打扰了，请问我6月初提交的退款申请，现在还没退成功。

我之前付款的银行卡已经停用了（换了卡号），请问我现在需要提供新的卡号吗？

望回复，谢谢！

Mike K ***@***.***> 于 2024年6月2日周日 13:57写道：

**Reply by MaxMKMax** [reply]
@YukieWu 
您好. 我们不需要卡号. 请您提供我支付单据和账号邮箱就可以了. 谢谢.

您可以在这里给我贴上支付单据吗?

**Reply by isluoting-sudo** [reply]
你好，我今天买了一年的专业版，但我发现它很慢且很多网站打不开，因此我申请退款。我的邮箱是15284172708@163.com，我通过支付宝付了352元，我想知道什么时候能完成退款。
![Uploading 微信图片_20250111223804.png…]()

**Comment by YukieWu** [comment]
***@***.***
谢谢！

Mike K ***@***.***> 于 2024年8月1日周四 01:19写道：
